### PR TITLE
move untar from entrypoint to dockerfile

### PR DESCRIPTION
**Pull request by ivanov-aleksander**
move untar from entry point  to dockerfile

### CODE REVIEW

**Comment by erszcz** [comment]
@michalwski As far as I understand, this causes the container to start up faster at the price of image size. I think we could live with that image size.

File: Dockerfile.member [hunk 1]
```diff
@@ -31,4 +31,5 @@ RUN apt-get update && apt-get install -y \
         odbc-postgresql && \
         apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+RUN tar xfz mongooseim.tar.gz -C /usr/lib || (echo "can't untar release" && exit 1)
```
Comment (erszcz) [inline Dockerfile.member]:
```suggestion
RUN (tar xfz mongooseim.tar.gz -C /usr/lib && rm mongooseim.tar.gz) || (echo "can't untar release" && exit 1)
```

**Comment by ivanov-aleksander** [comment]
I use this image in Kubernetes. And I couldn't mount my config maps because they will be rewritten when the container starts.

**Comment by ivanov-aleksander** [comment]
@michalwski 
I have change COPY commant to ADD - it will untar arhive and reduse size of final docker image

File: Dockerfile.member (hunk 1 above)
Comment (ivanov-aleksander) [reply Dockerfile.member]: I replace COPY to ADD command for tar archive

**Comment by erszcz** [comment]
@ivanov-aleksander Why not just mount the config map under `/memer`, where the container is designed to pick up configs from?

https://github.com/esl/mongooseim-kubernetes/blob/master/mongoose-sts.yaml#L44-L46

**Comment by michalwski** [comment]
That's neat! I was not aware docker's `ADD` command untars the archive. What's more, the size of the image is now smaller than the original one 119Mb :)

**Pull request merged**